storage/backend-test-suite/src/concurrent.rs

```diff
@@ -112,14 +113,17 @@ fn threaded_reads_consistent<B: Backend, F: BackendFn<B>>(backend_fn: Arc<F>) {
                 .to_owned()
         }
     });
-    let thr1 = thread::spawn(move || {
-        store
-            .transaction_ro()
-            .unwrap()
-            .get(IDX.0, TEST_KEY)
-            .unwrap()
-            .unwrap()
-            .to_owned()
+    let thr1 = thread::spawn({
+        let store = store.clone();
+        move || {
+            store
+                .transaction_ro()
+                .unwrap()
+                .get(IDX.0, TEST_KEY)
+                .unwrap()
+                .unwrap()
+                .to_owned()
+        }
```

Comment:
Maybe add an explicit `std::mem::drop(store);` after the two joins to be more explicit about where the store is dropped. Possibly, you won't need the clippy warning suppression then.

Reply:
I'm aware of this possibility (it's used in the other tests), but I wanted to keep this one different.